src/ensembl/src/content/app/species-selector/components/species-commit-button/SpeciesCommitButton.tsx

```diff
@@ -5,7 +5,7 @@ import {
   hasCurrentSpecies,
   canCommitSpecies
 } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
-import { commitSelectedSpecies } from 'src/content/app/species-selector/state/speciesSelectorActions';
+import { commitSelectedSpeciesAndSave } from 'src/content/app/species-selector/state/speciesSelectorActions';
```

Comment:
Are we going to be appending `AndSave` to names of actions that save data to web storage?

Reply:
We can make a convention out of this!

Reply:
What if we have an action that combines lots of different stuff: makes one or several requests, transforms data, saves transformed data and fires off another action? I'm a bit scared of enforcing naming conventions on actions.

Reply:
That's too much work for an action but as you say we can't rule it out. How about using this convention only for actions that do a simple save without any data transformation? Or maybe we can look into this later.

Reply:
> That's too much work for an action 

In the world of thunks — I don't think so. In the world of sagas (redux-saga), or epics (redux-observable), or other clever libraries (`redux-logic`, `redux-loop`) this additional logic would probably be delegated to those sagas or epics, etc.; but in the world of thunks, since thunks are actions, this complex work by necessity has to be done in an action.